nginx/nginx.conf

```diff
@@ -0,0 +1,17 @@
+upstream backend {
+    
+    server wordpress;
```

Comment:
let me make this task more interesting. What if I told you that the number of replicas of wordpress could be changed in time?

Reply:
That's a good one! Now my nginx.conf file is "replica-indepenent" 😄